_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -110,27 +105,25 @@ In this section, you will use Download Play to transfer the exploit data from th
 1. Wait a while (a percentage should be displayed on the **source 3DS**)
 1. If the exploit was successful, the **target 3DS** will have booted into the 3DS ROP xPloit Injector
   - If the exploit was not successful, power off the **source 3DS** and **target 3DS** and start again from the beginning of `Section III - kartdlphax`
-1. Press (X) to inject unSAFE_MODE
-1. If the injection was successful, the screen will turn green and the **target 3DS** will automatically power off
-  + If the screen turns red, power off the target 3DS and start again from the beginning of `Section III - kartdlphax`. If this doesn't work, ask for help at [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp))
 
-You will **not** need to use your **source 3DS** to complete any further steps on this guide. Any further steps should only be completed on the **target 3DS**.
-{: .notice--info}
-
-#### Section V - unSAFE_MODE
+## Section IV - Installing menuhax67
 
-{% include_relative include/exploit-usm.txt %}
+{% include_relative include/menuhax67-install.txt %}
 
-#### Section VI - Installing boot9strap
+#### Section V - Installing boot9strap
 
 {% include_relative include/install-boot9strap-safeb9sinstaller.txt %}
 {%- include_relative include/configure-luma3ds.txt %}
 
 {% include_relative include/luma3ds-installed-note.txt %}
 
-#### Section VII - Restoring WiFi Configuration Profiles
+#### Section VI - Removing menuhax67
+
+In this section, you will use the Homebrew Launcher to remove menuhax67, which will let you access the HOME Menu Settings option normally.
 
-{% include_relative include/remove-usm-slottool.txt %}
+{% include_relative include/launch-hbl-dlp.txt %}
+1. Launch menuhax_manager from the list of homebrew
+1. Select REMOVE menuhax67
```

Comment:
maybe we could add something about expected behaviour from this in a subbullet

Reply:
what is the expected behaviour lol